index.html

```diff
@@ -61,11 +61,11 @@
         <br>
         <div class="row" id="questionTwo">
             <div class="col-sm-1"></div>
-            <div class="col-sm-10" id="questions">
+            <div class="col-sm-10 questions" id="question_2">
                 <p>While attending the US Open with Jerry, George describes a woman as a "B.L.". What does B.L. stand for?</p>
                 <div class="custom-controls-stacked">
                     <label class="custom-control custom-radio">
-                        <input id="question2Answer1" name="radio-stacked" type="radio" class="custom-control-input">
+                        <input id="question2Answer1" name="radio-stacked" value="Brett Leonard" type="radio" class="custom-control-input">
```

Comment:
By adding a value to the radio button it makes it easier to pull the value off of the jquery object.